canvas_account_admin_tools/settings/local.py

```diff
@@ -57,3 +57,8 @@
    'selenium_grid_url': SECURE_SETTINGS.get('selenium_grid_url'),
    'use_htmlrunner': SECURE_SETTINGS.get('selenium_use_htmlrunner', True),
 }
+
+CONCLUDE_COURSES_URL = SECURE_SETTINGS.get(
+    'conclude_courses_url',
+    'https://icommons-tools.dev.tlt.harvard.edu/course_conclusion'
+)
```

Comment:
This will work, but you might want to discuss whether this could be hardcoded to point to the dev url.  Unit tests should be able to mock out a setting value if needed, but I could see an argument for allowing this to point to localhost when running the app locally.